lib/sensors/pressure.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/adc.hpp>
+
+namespace hyped::sensors {
+class Pressure {

```

Comment:
add a "TODOLater: test with hardware once we have it" above this

Reply:
also might be worth naming this `SuspensionPressure` (to differentiate from `CoolingPressure` which is for propulsion)